lib/util/variable.js

```diff
@@ -11,16 +11,21 @@
  * @returns {Boolean} True if the variable was found, false if not.
  */
 function findVariable(variables, name) {
-  var i;
-  var len;
-
-  for (i = 0, len = variables.length; i < len; i++) {
-    if (variables[i].name === name) {
-      return true;
-    }
-  }
+  return variables.some(function (variable) {
+    return variable.name === name;
+  });
+}
 
-  return false;
+/**
+ * Find and return a particular variable in a list
+ * @param {Array} variables The variables list.
+ * @param {Array} name The name of the variable to search.
+ * @returns {Object} Variable if the variable was found, null if not.
+ */
+function getVariable(variables, name) {
+  return variables.find(function (variable) {
```

Comment:
No worries; I pushed up https://github.com/yannickcr/eslint-plugin-react/commit/e332b08694de6271674bdf2182f03b399306ba82 to fix it